src/commands/run-tests-solidus-older.yml

```diff
@@ -7,6 +7,10 @@ parameters:
     default: '~/project'
 
 steps:
+  - test-branch:
+      branch: v4.0
+      rails_version: '~> 7.0'
+      working_directory: <<parameters.working_directory>>
```

Comment:
This has the unintended consequence of preventing us from testing extensions on versions of Ruby older than 3.x (due to Solidus 4.x's dependency on `solidus_sample`). This isn't necessarily a problem because Ruby 2.7 is no longer supported, but I just wanted to call it out, as some extensions still have their CI configured to run against that version.

Reply:
For an example of the failure you can see this workflow - https://app.circleci.com/pipelines/github/SuperGoodSoft/solidus_taxjar/803/workflows/a6eb7b21-8e7a-4bb1-acc7-ed1d20385b5d/jobs/1252

Reply:
Yes, that's something we faced in the past and we solved changing the ruby version the extension is tested against. I think that taking inspiration from a more recent configuration might be useful, eg: https://github.com/solidusio/solidus_auth_devise/blob/e82adb591fdc03a3f8ffd0cd31e6e0cc26e9f285/.circleci/config.yml. Let me know if this helps!

Reply:
Yeah, we've opted to do that now as well :+1: I posted here mostly for visibility since, technically all Solidus versions prior to 4.0 work with Ruby 2.7.